ncsa-teststand/AT/docker-compose.yml

```diff
@@ -7,17 +7,15 @@ services:
     container_name: atmcs-sim
     environment:
       - LSST_DDS_PARTITION_PREFIX=${SITE_PARTITION_PREFIX}
-      - LSST_DDS_RESPONSIVENESS_TIMEOUT=15s
```

Comment:
Are you sure you want to remove this? Isn't the default infinity?

Reply:
The 15s was when they were being run in SP mode. Otherwise, unless it's a daemon, it should be the default.